src/Factory/DomPdfFactoryInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusWishlistPlugin\Factory;
+
+use Dompdf\Dompdf;
+
+interface DomPdfFactoryInterface
+{
+    public function createNew(): Dompdf;
```

Comment:
Do we have any interface for this class?

Reply:
Dompdf is external package without provided interface